lib/foodcritic/rules.rb

```diff
@@ -826,3 +826,28 @@ def invalid_name(ast)
     end
   end
 end
+
+rule 'FC061', 'Valid cookbook version should be defined' do
+  tags %w{metadata}
+  metadata do |ast, filename|
+    ver = ast.xpath(%Q(//command[ident/@value='version']/descendant::tstring_content/@value))
+    if ver.nil? || ver == ''
+      [file_match(filename)]
+    else
+      pass = true
+      parts = ver.to_s.gsub(/\s|'|"/, '').split('.')
+      if parts.length > 1
```

Comment:
Probably okay to accept one or two parts and let it extend them, or at least that is pretty common and doesn't seem like a bad idea.